docs/src/app/components/pages/components/DatePicker/ExampleInline.js

```diff
@@ -8,6 +8,12 @@ const DatePickerExampleInline = () => (
   <div>
     <DatePicker hintText="Portrait Inline Dialog" container="inline" />
     <DatePicker hintText="Landscape Inline Dialog" container="inline" mode="landscape" />
+    <DatePicker
+      hintText="Keyboard Enabled Dialog"
+      container="inline"
+      mode="landscape"
+      keyboardEnabled={true}
```

Comment:
I have seen other places in the code where props like these are alphabetically sorted. Is that a convention? Should it be done here?